recipes/cluster-picker/cluster-picker.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
You might want to look at: https://bioconda.github.io/guidelines.html#java

There is a reference to a similar script in python that is maybe more readable and maintainable.